training/performance/README.md

```diff
@@ -161,15 +161,14 @@ Here is an overview of what features can help to either improve speed or save me
 | Method                   | Speed  | Memory |
 | :----------------------  | :----  | :----- |
 | Gradient accumulation    | Yes    | Yes    |
-| Gradient checkpointing   | Yes    | Yes    |
+| Gradient checkpointing   | Yes*   | Yes    |
 | Mixed precision training | Yes    | No     |
 | Batch size               | Yes    | Yes    |
 | Optimizer choice         | Yes    | Yes    |
 | DataLoader               | Yes    | No     |
 | DeepSpeed Zero           | No     | Yes    |
 | Flash Attention          | Yes    | Yes    |
-
-
+*It slows things down for the given batch size, but since it frees up a lot of memory, enabling a much larger BS, it actually improves the overall speed.
```

Comment:
```suggestion

\* Gradient checkpointing slows things down for the given batch size, but since it frees up a lot of memory, enabling a much larger BS, it actually improves the overall speed.
```